admin/public/js/components/erizo_list.jsx

```diff
@@ -0,0 +1,61 @@
+define(['pubsub', 'react', 'jsx!components/erizo_item'], function (PubSub, React, ErizoItem) {
+
+  var DATA_INTERVAL = 5000;
+
+  return React.createClass({
+    displayName: 'ErizoList',
+
+    getInitialState: function() {
+      return { erizos:[], stats: {}, showStats: false }
+    },
+
+    getDefaultProps: function() {
+      return { stats: {} };
+    },
+    
+    updateErizoJSData: function() {
+      var that = this;
+
+      PubSub.broadcast("ErizoAgent.getErizoJSInfo", null, function(resp) {
+        that.setState({erizos: resp.erizos});
+      });
+      
+      setTimeout(this.updateErizoJSData, DATA_INTERVAL);
+    },
+
+    componentDidMount: function () {
+      var that = this;
+      this.updateErizoJSData();
+      
+      PubSub.subscribe('stats', function(stat) {
+        var stats = that.state.stats;
+        stats[stat.pub] = stat;
+        that.setState({stats: stats});
+      });
+    },
+    
+    render: function() {
+      var that = this;
+      var createItem = function(item) {
+        return (
+            <ErizoItem item={item} key={item.id} stats={that.state.showStats ? that.state.stats : {}}/>
```

Comment:
It's probably not a problem here, but it reminded me: one thing about the `key` prop is, if React goes to do DOM updates and would normally just update a child component, but the before-and-after components have different `key`s, React will destroy the old one and recreate the new one.